eclipse/sw360
# fix(UI): Clearing Request bug fix and improvements

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
 * *Issue #799*, changes w.r.t clearing request functionality.
 * No changes in dependencies.
 * All the `CR` under a user's department should be listed in `open/closed clearing request` tab.
 * Re-Open `Closed` or `Rejected` clearing request for a project.
 * Only Users with `WRITE` access to a project can create or re-open the Clearing Request.
 * `Rejected` and `Closed` Clearing Request will be displayed under `Closed Clearing Request` tab.
 * Trigger an `email` for adding a comment in `Clearing Request`.
 * All Email notifications will be sent only to `Requesting User` and `Clearing Team` irrespective of subscription.


Issue: #799 

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
* Check only users with `WRITE` access to a project can create a Clearing Request,
* All `Rejected` and `Closed` Clearing Request will be displayed under `Closed Clearing Request` tab.
* Users with `WRITE` access to a project can create or re-open the Clearing Request.
* All the `CR` under a user's department is listed in `open/closed clearing request` tab.
* Check that Emails are triggered for adding a comment in CR.
* No additional tests implemented.

### Checklist
Must:

Signed-off-by: akapti <abdul.mannankapti@siemens.com>
